workflow-language/src/main/java/com/symphony/bdk/workflow/swadl/v1/Event.java

```diff
@@ -13,7 +13,6 @@
 import com.symphony.bdk.workflow.swadl.v1.event.RequestReceivedEvent;
 import com.symphony.bdk.workflow.swadl.v1.event.RoomCreatedEvent;
 import com.symphony.bdk.workflow.swadl.v1.event.RoomDeactivatedEvent;
-import com.symphony.bdk.workflow.swadl.v1.event.RoomMemberDemotedFromOwnerEvent;
```

Comment:
Can you please change the field type in ligne 79 to RoomMemberDemotedFromOwnerEvent?

Reply:
sure.